pkgs/applications/networking/ids/bro/default.nix

```diff
@@ -2,16 +2,20 @@
 , geoip, gperftools, python, swig }:
 
 stdenv.mkDerivation rec {
-  name = "bro-2.5.5";
+  name = "bro-2.6.2";
 
   src = fetchurl {
     url = "https://www.bro.org/downloads/${name}.tar.gz";
-    sha256 = "1kvkiq8jjsqryry9jd4vw45pbfb46jly988mq62mv4sd1fqsxwhq";
+    sha256 = "19n0xai1mndx2i28q9cnszam57r6p6zqhprxxfpxh7bv7xpqgxkd";
   };
 
   nativeBuildInputs = [ cmake flex bison file ];
   buildInputs = [ openssl libpcap perl zlib curl geoip gperftools python swig ];
 
+  # Indicate where to install the python bits, since it can't put them in the "usual"
+  # locations as those paths are read-only.
+  cmakeFlags = [ "-DPY_MOD_INSTALL_DIR=${placeholder "out"}/${python.sitePackages}" ];
```

Comment:
Most Zeek/Bro installation come with the *auxiliary tools*, like `bro-cut`. Thus, the `"-DINSTALL_AUX_TOOLS:BOOL=true"` cmake flag should be added.